releasenotes/notes/0.25/dag-substitute-node-propagate-condition-898052b53edb1f17.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-features:
+features_circuits:
```

Comment:
I would probably put this under transpiler too, because at least in my mind the `DAGCircuit` api is for the transpiler and not really for circuit users.

Reply:
Ah that's an interesting point. There are probably quite a few places where I'd need to make that change 😄.